### ISSUE TITLE
Release v1.2.0

### ISSUE THREAD

**Issue by orangemug** [maputnik/editor#288]
A new version has been released to <https://maputnik.github.io/releases/v1.2.0-beta2>

If anyone has time to give it a try, please report anything broken that wasn't broken in the previous version, or respond 'It looks good to me!'

CC @pjsier @gregorywolanski @pathmapper @keum

**Comment by keum** [comment]
@orangemug Thanks for updating and sharing with us!

One thing I noticed in Export  function (which I use a lot to showcase/share web map version) is that Save to Gist under Export Style stopped working while back (Gist prevented anonymous post) old version and v1.2.0 also won't work either. Maybe it needs to be removed? Not sure if I'm doing wrong from my end. 

Also after I entered both Tilehosting key and Mapbox key then export style, looking at style.json file, still won't show API key in the style. I can get around by editing it and adding it myself but would be nice to have that working . 
```
  "sources": {
    "openmaptiles": {
      "type": "vector",
      "url": "https://free.tilehosting.com/data/v3.json?key={key}"
```
So far I haven't come across anything not working yet. others might disagree though. 
Thanks for all your work and there are many really appreciate 🙏 continually to improve maputnik editor. thanks!

**Comment by orangemug** [comment]
> Save to Gist under Export Style stopped working while back

Yeah apologies GitHub scrapped support for anonymous Gists. You're right we should remove it until it's fixed, I'll do that this release.

> Also after I entered both Tilehosting key and Mapbox key then export style, looking at style.json file, still won't show API key in the style. I can get around by editing it and adding it myself but would be nice to have that working

That seems reasonable, could you create a new issue.

**Comment by keum** [comment]
@orangemug i'll do that. thanks

**Comment by orangemug** [comment]
Released v1.2.0, see blog post <https://maputnik.github.io/blog/2018/05/08/release-v1.2.0>

Congrats to all that contributed!
